static/html/5xx.html

```diff
@@ -5,14 +5,13 @@
         <title>Zulip - 500 internal server error</title>
         <base href="/static/webpack-bundles/">
 
+        <!--# if expr="$status = 502" -->
         <meta http-equiv="refresh" content="60;URL='/'">
+        <!--# endif -->
```

Comment:
Can you explain how this mechanism works?

Reply:
(Maybe with a link to the relevant part of the nginx documentation?)

Reply:
So once we have enabled SSI we can access variables/files from nginx backend. In `nginx.conf` we have mapped status codes with their error message. So we are using status code sent via nginx and printing it's code along with the message.

Here we are using the status code sent via nginx and only reloading the page if it was a 502 error code. Also, why the if condition? It makes no sense to reload the site until and unless it was a bad gateway.

SSI Guide: https://www.cambus.net/nginx-and-server-side-includes/
SSI Documentation: https://nginx.org/en/docs/http/ngx_http_ssi_module.html